CHANGELOG.md

```diff
@@ -1,14 +1,406 @@
+- Violet Outskirts was also given a spooky event. Take the Sylph Scope there for an evert mon.
+- Removed Celadon Outskirts.
+- Dynamic level scaling based on badge number was included for both wildmons and trainers in certain locations.
+    For Wildmons: Hidden grottoes, all optional areas, and the Safari Zone.
+    For Trainers: Route Leaders, and alls trainers in the optional routes.
+- Three-Segment Dudunsparce more common during swarms.
+- Holding B on cycling road hits the brakes on the bike.
+- You can now call Elm to know how your lead party mon evolves. You can also call Lyra to find out when your mon will learn its next move (but it will not specify the move). There is also a hint that Lyra calls are useful now.
+- New heal spot right before Victory Road.
+- Cerulean City Slowpoke is a Slowbro now.
+- The Abra man in the League now stays until you can fly to Vermillion City.
+- Added a second Rawst Berry tree in Violet Outskirts.
+- Fixed the area where the Go-Goggles are found to make it more obvious where to surf. And added an NPC to hint to where it is.
+- NPC trade changes (such as one for Galarian Weezing and changing the one in Violet so that it's a Poliwag for a Voltorb).
+- Lucky Number is now Daily, not Weekly.
+- Moved and added, or removed/replaced some cameo trainers. These include Piers, Katy, Thorton and Melony added, while Skyla was replaced with Marlon and Anabel was moved to the outside of Battle Tower.
+- Many added pokemon are now available in the wild.
+- Alolan Exeggutor now blocks paths instead of Vileplume.
+- Added a nurse's office to Celadon University.
+- Added an Endure move tutor.
+- Added Kimono Girls Mako and Mina.
+- Nature Changing NPC with mint leaf tea.
+- Added a bunch of pokemon journals.
+- Several NPC interactions (haircuts and the friendship option for Shrine Maiden Rei) now exclusively give affection.
+- More NPC hints about all kinds of features and items.
+- Olivine Lighthouse shines at night.
+- Goldenrod Museum is now open for business.
+- Plays a sound when the Rocket member takes a gate toll.
+- Toned down poison screen effect for accessibility.
+- Vulcandth added to the NPCs in the dev room.
+- Added sand footprints. 
+- Imported some text stuff from Prism.
+- Changed the abbreviation of Speed to Spe instead of Spd as it was ambiguous.
+- Remove roam mon anti-oscillation feature. (In Gen II, and ONLY in Gen II, if you return to the map you previously visited, roamers are forbidden from jumping there.)
+- Weather effects, including rain (with lightning 25% of the time), sandstorm, and snow, now occur in the overworld on certain maps on certain days.
+
+_Miscellaneous:_
+- Feature Glaceon in the intro to reflect Rangi's profile picture.
+- Analogue Pocket support for the non-RTC version.
+- Updated credits for sprite work (gmerc, Farore) and contributions.
+
+### Fixes
```

Comment:
Maybe mention that some of **_these_** fixes may be intra-version fixes, and may not be all carried over from v3.0.0-beta. I know it be too difficult to differentiate, so a simple disclaimer is fine.